src/test/tests/databases/reopen.py

```diff
@@ -82,7 +85,7 @@ def TestLength(testname):
 # Returns whether all files in the list are in the current directory.
```

Comment:
You might want to update this comment to say in the run directory instead of the current directory.